web/WebDB.php

```diff
@@ -295,6 +295,13 @@ public static function getWalletStats() {
       $wallets[ $coin ][ $exid ][ 'opportunities' ] = intval( $row[ 'uses' ] );
       $wallets[ $coin ][ $exid ][ 'change' ] = floatval( $balance  - self::getHistoricBalance( $coin, $exid ) );
       $wallets[ $coin ][ $exid ][ 'trades' ] = intval( $row[ 'trades' ] );
+      if ($coin=="BTC") {
+        $wallets[ $coin ][ $exid ][ 'balance_BTC' ] = floatval(formatBTC($balance));
+        $wallets[ $coin ][ $exid ][ 'change_BTC' ] = floatval( ($balance  - self::getHistoricBalance( $coin, $exid)));

```

Comment:
The `change_BTC` stuff doesn't make sense really, let me explain.  As the bot watches the market, it computes a desired balance for each altcoin currency, which means the amount of altcoins it wants to hold.  The "change_foo" fields for the altcoins denote the changes in this value.

As you can see, aggregating this value across all altcoin values doesn't really make a whole lot of sense, so let's remove the `change_BTC` parts and keep the `balanace_BTC` parts?